### DISCUSSION TITLE
Not a bug just a question. How do I update forge itself to the latest version?

### DISCUSSION THREAD

**Discussion by Lindsaydoodle** [IzzelAliz/Arclight#690] [Q&A]
Not a bug just a question. How do I update forge itself to the latest version?

Some mods do not run because of a low version of forge need to somehow update but I do not know how to do it. Please someone write detailed instructions on how to update forge.

thanks in advance

**Comment by Yizhouuu** [comment]
did you find a solution?

**Comment by Fix717** [comment]
ответ не найден?

**Comment by qyl27** [comment]
Have you read our document? 
https://wiki.izzel.io/s/arclight-docs/doc/faq-ApcZ25PXVT#h-could-i-use-the-custom-forgeneoforge-version
